--- sys_linux/g77_elf.conf.pius	Fri Mar 28 01:31:27 1997
+++ sys_linux/g77_elf.conf	Sat Nov 13 04:00:22 1999
@@ -69,13 +69,13 @@
 # The library-specification flags to use when linking normal pgplot
 # demo programs.
  
-   LIBS="-L/usr/X11/lib -lX11"
+   LIBS="-L/usr/X11R6/lib -lX11"
 
 # Optional: Needed by XMDRIV (/xmotif).
 # The library-specification flags to use when linking motif
 # demo programs.
  
-   MOTIF_LIBS="-lXm -lXt -lX11"
+   MOTIF_LIBS="-L/usr/X11R6/lib -lXm -lXt -lX11"
 
 # Optional: Needed by TKDRIV (/xtk).
 # The library-specification flags to use when linking Tk demo programs.
@@ -92,12 +92,12 @@
 # Optional: Needed on systems that support shared libraries.
 # The name to give the shared pgplot library.
  
-   SHARED_LIB="libpgplot.so"
+   SHARED_LIB="libpgplot.so.5.2.0"
 
 # Optional: Needed if SHARED_LIB is set.
 # How to create a shared library from a trailing list of object files.
  
-   SHARED_LD="gcc -shared  -o $SHARED_LIB"
+   SHARED_LD="g77 -shared  -o libpgplot.so.5.2.0  -Wl,-soname,libpgplot.so.5  -lc -f2c"
 
 # Optional:
 # On systems such as Solaris 2.x, that allow specification of the
